ev3dev2/motor.py

```diff
@@ -1924,6 +2037,181 @@ def get_speed_steering(self, steering, speed):
         return (left_speed, right_speed)
 
 
+class MoveDifferential(MoveTank):
+    """
+    For all methods speed can be an integer or a SpeedInteger object
```

Comment:
Can you add a full documentation comment here explaining what the class is?

Reply:
Yeah will write something up.